seda.muradyan/repo/Lesson_3/exercise_13/exercise_13.py

```diff
@@ -0,0 +1,14 @@
+# Home work lesson 3/13
+
+price = float(input ("Enter price of the meal: "))
+
+taxPracent = 22
```

Comment:
CSW: Bad variable name. Change it to the pythonic way. Also, use English words (percent).